opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -331,6 +340,22 @@ enum DataPointFlags {
   // Bits 2-31 are reserved for future use.
 }
 
+// GaugeSubType defines valid gauge "sub-types", which are used to describe
+// "special" kinds of gauges which have additional properties. These sub-types
+// generally apply to metrics which originate from protocols which support
+// these additional types, but are represented as gauges in OTLP.
+enum GaugeSubType {
+  // GAUGE_SUBTYPE_NONE is the default GaugeSubType, which represents a gauge
+  // that does not have a sub_type.
+  GAUGE_SUBTYPE_NONE = 0;
```

Comment:
@tigrannajaryan just want to confirm that this doesn't collide with any of the work you recently did to normalize enum values. For other enums in OTLP, we have the following patterns:
- `{ENUM_NAME}_UNSPECIFIED`
- `{ENUM_NAME}_UNSET`
- `{ENUM_NAME}_DO_NOT_USE`
I think its fine but wanted to double check since we're stable now.